readme.md

```diff
@@ -185,6 +185,7 @@ module.exports = {
 | [sort-intersection-types](https://perfectionist.dev/rules/sort-intersection-types)       | Enforce sorted intersection types             | 🔧  |
 | [sort-jsx-props](https://perfectionist.dev/rules/sort-jsx-props)                         | Enforce sorted JSX props                      | 🔧  |
 | [sort-maps](https://perfectionist.dev/rules/sort-maps)                                   | Enforce sorted Map elements                   | 🔧  |
+| [sort-modules](https://perfectionist.dev/rules/sort-modules)                             | Enforce sorted modules and namespaces         | 🔧  |
```

Comment:
do you finally want to mention namespaces? maybe you should explain in the documentation how they will be handled?

Reply:
@OlivierZal I agree that it's not mentioned in the documentation as I thought it would be a bit confusing:
the rule sorts `modules` and `namespaces` but not together:

```ts
function a() {}
 function b() {} // We are sorted!


namespace B {
	function a() {}
    function b() {} // We are sorted!
}

namespace A { // namespaces are not sorted together
	function a() {}
    function b() {} // We are sorted!
}
```

So I think we could say it sorts modules and namespaces **content** instead?

Reply:
Looks good to me

Reply:
Updated here f954b6d1e91009e58e96e2d3f2d74a96306b2b4c